resources/keywords/deploy_instance_dev.resource

```diff
@@ -63,7 +63,7 @@ User Selects ${databaseProvider} Tile Under Cloud Hosted Database
     Element Should Be Visible    ${hdrCloudHostDBScreen}
 
 User Selects Database Instance For The Provider Account
-    Wait Until Page Contains Element    ${drpDwnDevProvAccount}     timeout=10s
+    Wait Until Keyword Succeeds     10x     2s      Wait Until Page Contains Element    ${drpDwnDevProvAccount}
```

Comment:
Can we change the 10x to _time in seconds_ instead? With this current implementation, if the element is not available on the page the keyword will fail after 70 secs, which is a lot a waiting. We can maybe change the 10x to 20s or 30s. Please let me know what you think